app/Livewire/GlobalSearch.php

```
@@ -62,7 +71,63 @@ public static function clearTeamCache($teamId)
+            $this->loadCreatableItems();
+            $this->searchResults = [];
+
+            // Check for sub-commands like "new project", "new server", etc.
+            // Use original query (not trimmed) to ensure exact match without trailing spaces
+            $this->autoOpenResource = $this->detectSpecificResource(strtolower($this->searchQuery));
+        } else {
+            $this->isCreateMode = false;
+            $this->creatableItems = [];
+            $this->autoOpenResource = null;
+            $this->search();
+        }
```

PHPMD (2.15.0)

The method updatedSearchQuery uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)